test/topology/util.py

```diff
@@ -448,3 +448,13 @@ async def new_materialized_view(cql, table, select, pk, where, extra=""):
         yield mv
```

Comment:
Regarding commit message for " raft: Add a test to check log size after truncation "

contains too many "when":

"The test checks, that when snapshot_trailing_size parameter is taken
in to consideration when the log system table is truncated."

The first "when" should be gone

Also there should be no "," after "checks"

Reply:
Fixed.